client/src/components/users/UserInformation.vue

```diff
@@ -61,6 +72,10 @@ const props = defineProps({
     type: Boolean,
     default: true,
   },
+  activeFields: {
```

Comment:
Not a big fan of those updates. The API is a bit convoluted, it doesn't make much sense, you can get weird cases like no active fields at all.
If someone wants an input to get a device name, their first thought shouldn't be `I'll use the UserInformation component and deactivate name and email`, it should be `I'll create an input`.